2-structured-data/config.js

```diff
@@ -47,11 +47,6 @@ nconf
     // This is the id of your project in the Google Cloud Developers Console.
     GCLOUD_PROJECT: '',
```

Comment:
Ok, but this should have been GOOGLE_CLOUD_PROJECT - we don't want to abbreviate this

Reply:
All the kokoro scripts, test scripts and app code are set to use this environment variable. I will double check to make sure that app is still runnable after changing it to `GOOGLE_CLOUD_PROJECT`.

Reply:
It's fine not changing it, but for the future it should change.